src/reducers/projects.js

```diff
@@ -83,6 +83,21 @@ export function reduceRoot(stateIn, action) {
             projects.get(currentProjectKey),
           );
         }
+      case 'FOCUS_LINE':
+        {
+          const currentProjectKey =
+            stateIn.getIn(['currentProject', 'projectKey']);
+
+          if (isNil(currentProjectKey)) {
+            return stateIn;
+          }
```

Comment:
When would a `FOCUS_LINE` action occur with a nil `currentProjectKey`? My impression is that the only time there's not a current project key is when the app is loading.